### ISSUE TITLE
[llvm] llvm-config configuration file path is incorrect

### ISSUE THREAD

**Issue by etheory** [microsoft/vcpkg#10559]
**Host Environment**
- OS: OS Name Microsoft Windows 10 Home, Version	10.0.18362 Build 18362
- Compiler: Visual Studio Community 2019 16.5.1

**To Reproduce**
`./vcpkg install llvm:x64-windows`
work through the innumerable errors and requirements for manual renaming, wait over 4 hours, then....

**Failure logs** 
After installing, if I run:
`llvm-config.exe --libs`

It produces the following:
```
C:\Users\etheory>Z:/Luke/Programming/Graphics/vcpkg/installed/x64-windows/tools/llvm/llvm-config.exe --libs
llvm-config: error: component libraries and shared library

llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMDemangle.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMSupport.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMBinaryFormat.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMCore.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMBitReader.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMDebugInfoMSF.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMDebugInfoCodeView.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMMC.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMMCParser.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMObject.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMOption.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMLibDriver.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMDebugInfoDWARF.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMDebugInfoPDB.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMSymbolize.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMDlltoolDriver.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMProfileData.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMCoverage.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMRuntimeDyld.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMAnalysis.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMTarget.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMExecutionEngine.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMMCJIT.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMX86Utils.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMX86AsmPrinter.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMX86Info.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMMCDisassembler.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMX86Desc.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMBitWriter.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMTransformUtils.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMAggressiveInstCombine.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMInstCombine.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMScalarOpts.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMCodeGen.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMAsmPrinter.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMSelectionDAG.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMGlobalISel.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMX86CodeGen.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMX86AsmParser.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMX86Disassembler.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMObjectYAML.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMInterpreter.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMOrcJIT.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMTableGen.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMMCA.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMTextAPI.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMLineEditor.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMOptRemarks.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMXRay.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMFuzzMutate.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMWindowsManifest.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMAsmParser.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMMIRParser.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMIRReader.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMLinker.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMVectorize.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMInstrumentation.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMipo.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMObjCARCOpts.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMPasses.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMLTO.lib
llvm-config: error: missing: Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\tools\lib\LLVMCoroutines.lib
```

Because the ACTUAL location is: `Z:\Luke\Programming\Graphics\vcpkg\installed\x64-windows\lib`

So a 4.773 hour install of llvm has produced a result that can't be found by anything that calls the usual method to find the install.... Extremely frustrating. I was trying to build OpenShadingLanguage, and during the setup, it prints exactly the same output as above, since llvm via vcpkg gets configured incorrectly.

**Additional context**
....

**Comment by JackBoosY** [comment]
Related issue: #9622. 
I wish this issue will be fixed in #10295.

**Comment by etheory** [comment]
Thanks @JackBoosY this is very unfortunate due to the incredibly long amount of time it takes to install, so yes, I REALLY hope it gets fixed soon. Thanks for following up though! Is there a PR happening for the renaming bug? If so, any ideas on how long that will be before it is fixed?

**Comment by JackBoosY** [comment]
@etheory Since the new version of llvm uses a new configuration method, we may not be bothered by this issue again.

**Comment by etheory** [comment]
@JackBoosY will this be back-ported to llvm-8 or llvm-9 for older projects that need those versions? Or will I have to use only 10 now when this gets fixed?

**Comment by JackBoosY** [comment]
I think it's hard to say, we don't accept downgrading the port, and due to the modification of the configuration system, changes in llvm 10.0 will be difficult to incorporate into the lower version.

**Comment by yurybura** [comment]
@etheory @JackBoosY 
Merged PR #10295 fixes this issue. Please check.

**Comment by JackBoosY** [comment]
Close this issue via #10295 merged.
